org.musicbrainz.Picard.json

```diff
@@ -214,7 +214,7 @@
                 {
                     "type": "git",
                     "url": "https://github.com/metabrainz/picard.git",
-                    "commit": "8444f01734656aed661f5f52ce6b62c517835347" /* 2.1.0 */
+                    "tag": "release-2.1.1"
```

Comment:
Please use **both** the `tag` and `commit` property, to make the build reproducible. (in case they force-push a change to the tag)